README.md

```diff
@@ -53,6 +53,32 @@ some_metric{foo="bar"} 1
 
 Secret verification can be completed using sending get-parameter `?secret=XXX` or HTTP header `X-SECRET`.
 
+### Formatters
+
+Out of the box this package provides `json` and `prometheus` metrics formats. Feel free to write your own formatters.
+
+#### Prometheus formatter
+
+Prometheus formatter uses some constants for metric's values(`Nan`, `+Inf`, `-Inf`) and types(`COUNTER`,`SUMMARY`,`GAUGE`,`HISTOGRAM`, `UNTYPED`). You can find it in
```

Comment:
In a one line, please (without line breaks)

Reply:
Fixed